_guidelines-v4/10-analysisharm/01-analysis/01-analysisdescribingrelationships.md

```diff
@@ -6,28 +6,30 @@ version: "v4"
 
 The relationships between event elements, such as note, chord, and rest, are the basic material of musical analysis; the attributes described below ensure a closed network of these relations and provide the opportunity to record data useful for common analytical tasks. In the context of a formal analysis, for instance, the attributes presented here can be useful in the capture information about the structure of a musical work.
 
-The shared module offers several attributes in the {% include link att="linking" %} class for the description of basic relationships:
+MEI offers several attributes in the {% include link att="linking" %} class for the description of basic relationships:
 
 {% include desc atts="att.linking/copyof
+att.linking/sameas
 att.linking/corresp
 att.linking/next
 att.linking/prev
-att.linking/sameas
+att.linking/precedes
+att.linking/follows
 att.linking/synch" %}
```

Comment:
Is this order related to anything? Wouldn't it be better to have it in alphabetical order?

Reply:
This was requested by someone (apologies for not knowing who it was right now…). I think this order makes sense, in that it order the attributes by relevance / frequency of usage. Yes, we can argue if `@copyof` should really go first or `@sameas`, but I think it's clear that together, they should precede `@next` and `@prev`, etc. So this is intentional, and I hope it's more instructive as well… 

Reply:
I am fine if it makes sense for you ;)